pkg/cache/v3/resource_test.go

```diff
@@ -57,15 +58,15 @@ var (
 )
 
 func TestValidate(t *testing.T) {
-	assert.NoError(t, testEndpoint.Validate())
-	assert.NoError(t, testCluster.Validate())
-	assert.NoError(t, testRoute.Validate())
-	assert.NoError(t, testScopedRoute.Validate())
-	assert.NoError(t, testVirtualHost.Validate())
-	assert.NoError(t, testListener.Validate())
-	assert.NoError(t, testScopedListener.Validate())
-	assert.NoError(t, testRuntime.Validate())
-	assert.NoError(t, testExtensionConfig.Validate())
+	require.NoError(t, testEndpoint.Validate())
+	require.NoError(t, testCluster.Validate())
+	require.NoError(t, testRoute.Validate())
+	require.NoError(t, testScopedRoute.Validate())
+	require.NoError(t, testVirtualHost.Validate())
+	require.NoError(t, testListener.Validate())
+	require.NoError(t, testScopedListener.Validate())
+	require.NoError(t, testRuntime.Validate())
+	require.NoError(t, testExtensionConfig.Validate())
```

Comment:
Nit - `assert` is correct here. If the linter insists on this change, we can live with it though :)

Reply:
Evey assert.NoError to require.NoError change was required by the linter.`